### PR TITLE
Global

### PR DESCRIPTION
**Pull request by alicespringfeld**
I've added variables to CSS and created a layout for the Web-App.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,8 +3,9 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <link rel="stylesheet" href="style.css" />
```
Comment (lajanzen) [inline index.html:6]: We import the CSS file now via JS - we'll discuss this later today :) 

```suggestion
```

**Review by lajanzen — CHANGES_REQUESTED** [review]
🚀

**Review by markobrkovic — APPROVED** [review]
Looks good 💯😄

File: index.html [hunk 2]
```diff
@@ -3,8 +3,9 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite App</title>
+    <title>Rick and Morty</title>
```
Comment (leonardpaulus) [inline index.html:8]: You have good  👀 :)

**Review by Ziedlassoued — APPROVED** [review]
lgtm 👍

**Pull request merged**